### PR TITLE
Update WS Agent debug instructions for Openshift/Kubernetes

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Add more instructions on debugging WS-Agent on Kubernetes/Openshift through port-forwarding

//TODO add instruction to retrieving OpenShift pod name with namespace-per-workspace strategy

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issue/11682

### CODE REVIEW

File: src/main/pages/developer-guide/workspace-agent.adoc [hunk 1]
```diff
@@ -17,27 +17,61 @@ folder: developer-guide
...
 image::wsagent/wsagent-debug-docker.png[]
 
-Local OKD:
+#Kubernetes / Openshift
+
+To connect with debugger from outside of the cluster, use Port Forwarding to forward port from your workspace Pod to the port on your localhost.
+
+Get list of pods and find the name of your workspace pod.
+```
+kubectl get pods --all-namespaces
```
Comment (sleshchenko) [inline src/main/pages/developer-guide/workspace-agent.adoc]: won't output of this command contains info about namespace as well? So, we can avoid executing one more command with describing pod

File: src/main/pages/developer-guide/workspace-agent.adoc [hunk 2]
```diff
@@ -17,27 +17,61 @@ folder: developer-guide
...
+Use this to get the namespace of your <POD_NAME> pod.
+```
+kubectl describe pod <POD_NAME> #
+```
+
+For Kubernetes:
 
-Use following command to retrieve cluster IP to use as a host (replace `wsId` with ID of your workspace):
+```
+# replace <JPDA_PORT> with port of your configured JPDA Tomcat
```
Comment (sleshchenko) [inline src/main/pages/developer-guide/workspace-agent.adoc]: Env var with brackets here `<>` but without in three next instruction. Please do in the same way

File: src/main/pages/developer-guide/workspace-agent.adoc [hunk 3]
```diff
@@ -17,27 +17,61 @@ folder: developer-guide
...
+```
+kubectl get pods --all-namespaces
+```
+
+Use this to get the namespace of your <POD_NAME> pod.
+```
+kubectl describe pod <POD_NAME> #
+```
+
+For Kubernetes:
```
Comment (sleshchenko) [inline src/main/pages/developer-guide/workspace-agent.adoc]: Should not it be moved before line # 40 
> Get list of pods and find the name of your workspace pod.

?

File: src/main/pages/developer-guide/workspace-agent.adoc (hunk 2 above)
Comment (mkuznyetsov) [reply src/main/pages/developer-guide/workspace-agent.adoc]: ok

File: src/main/pages/developer-guide/workspace-agent.adoc (hunk 3 above)
Comment (mkuznyetsov) [reply src/main/pages/developer-guide/workspace-agent.adoc]: ok

File: src/main/pages/developer-guide/workspace-agent.adoc (hunk 1 above)
Comment (mkuznyetsov) [reply src/main/pages/developer-guide/workspace-agent.adoc]: ty

**Review by sleshchenko — APPROVED** [review]
LGTM

**Review by rkratky — APPROVED** [review]

**Pull request merged**